ampersand-select-view.js

```diff
@@ -90,7 +91,13 @@ SelectView.prototype.render = function () {
         }.bind(this));
     }
 
+    // Message DOM nodes
+    this.mContainer = this.el.querySelector('[data-hook~=message-container]');
```

Comment:
I'm not sure this is the best idea. I know it looks more efficient but in my experience third party add-ons like typeaheads and other jquery-ui or bootstrap-type libs sometimes clobber (remove and recreate) UI elements, which would make this stop working.